config/locales/server.en.yml

```diff
@@ -55,6 +55,9 @@ en:
         title: After post cook
         description: When a post content is cooked the automation will be triggered
     scriptables:
+      llm_triage:
```

Comment:
Just a note here, I’m not sure if AI stuff should be leaking into this plugin? Can this plugin perhaps expose APIs that can be registered from discourse-ai instead?

Reply:
look my comment above allan, there's already one: https://github.com/discourse/discourse-automation/pull/217#discussion_r1340962110

Reply:
Oops sorry didn’t see that one.

Reply:
no worries